poetry.lock

```diff
@@ -91,15 +91,12 @@ yaml = ["pyyaml"]
 
 [[package]]
 name = "bcrypt"
-version = "3.2.2"
```

Comment:
I don't think we need to re-gen the lock file in this PR

Reply:
undid this.